Assets/Scripts/Orbit.cs

```diff
@@ -58,8 +60,10 @@ private void FixedUpdate()
 
         //if (!Input.GetButton("Fire2") || _player.transform.parent != _self) return;
         
-        //InputAction replaces "Input.GetButton("Example") and holds a bool
-        if (!slingshotAction.triggered || _player.transform.parent != _self) return;
+        // haven't selected a star
+        if (!_starToGo  || _player.transform.parent != _self) return;
+        ////InputAction replaces "Input.GetButton("Example") and holds a bool
+        // if (!slingshotAction.triggered || _player.transform.parent != _self) return;
```

Comment:
There's already a bool that controls when the slingshot starts (_launchBegan); either remove the original implementation and bundle the functionality in, or use the originally implementation and add these conditions.

Also, not sure if these conditions are even necessary since there's a null check earlier in the code, have to look at it more.